examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -153,6 +153,7 @@ struct event {
   char comm[TASK_COMM_LEN];
   uint8_t error_code;
   uint8_t stack_status;
+  int kernel_stack_id;
```

Comment:
This type should be explicit width because it's shared with the driver and both undergo compilation differently.